### ISSUE TITLE
No intellisense for typings even with jsconfig.json created

### ISSUE THREAD

**Issue by EthraZa** [Microsoft/vscode#1081]
I have **typings/extjs/ExtJS.d.ts** and **jsconfig.json** at root level of my Sencha ExtJS 6 project folder:
~~~
{
    "compilerOptions": {
        "module": "commonjs",
        "target": "ES5",
        "outDir": "out",
        "noLib": true,
        "sourceMap": true
    },
    "exclude": [
        "node_modules",
        "ext",
        "overrides",
        "resources",
        "sass",
        "build",
        "packages",
        "help"
    ]
}
~~~

When I open any js file I just get a popdown Warning saying:
~~~
Sorry, but there are too many JavaScript source files for VS Code. Consider using the exclude-property in jsconfig.json.
~~~
And no IntelliSense.

I have another web project that is not a Sencha MVC project, but uses ExtJS 4 on js files. In this project I get no warnings, but I get no IntelliSense as well.

**Comment by jrieken** [comment]
There an issue with `exclude` properties not being honoured property. It got fixed with the work on #703

**Comment by EthraZa** [comment]
I see. I understand it is two distinct issues so.
1- Is too many files to deal with because ignore is being... ignored.
2- Is no IntelliSense even in small projects where ignore can be ignored. (What makes me create this issue)

**Comment by jrieken** [comment]
1. We cannot handle large amount of files in web workers - Will be fixed by adopting project Salsa asap Microsoft/TypeScript#4789
2. Failing to handle the `exclude`-property correctly leads to large project (node_modules et al). That is fixed in master (#703)
